### ISSUE TITLE
Automate setup of OE Admin Set, Collection Type and Collection

### ISSUE THREAD

**Issue by eporter23** [emory-libraries/dlp-selfdeposit#392]
For this ticket, we want to ensure that if we do a manual wipe and restore of our application's data, we can streamline the steps needed to ensure that our customizations related to auto-assigning works deposited by non-Admins go into the correct Admin Set (and workflow) as well as the correct Collection.

This is a followup to bug ticket #378. We discussed looking at this as a rake task (with a separate ticket to try to incorporate this into Ansible).

As a reference, please see the following in oe24-test:

- [x] Admin Set: OpenEmory Deposits with Workflow
- [x] Collection Type: Library Collection
- [x] Collection: OpenEmory

Ideally, we could either run this manually only if we do a reset of data, or, the rake task will check to make sure the necessary assets/configurations exist before recreating them.

**Comment by eporter23** [comment]
We may also need to automate the creation of the 3 permission groups that are assigned to the OpenEmory collection instance. Those groups are OE_Manager, OE_Depositor, OE_Viewer in addition to the "admin" group which should exist by default. 

<img width="460" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBL1BwQlE9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--200de776986ee8de2fade2193af308595ae8dba1/Screenshot%202024-07-17%20at%2011.26.52%E2%80%AFAM.png" alt="Screenshot 2024-07-17 at 11.26.52 AM.png" />

**Comment by alexBLR** [comment]
rake selfdeposit:publications:setup_admin_set_collection @jcrompton42

**Comment by eporter23** [comment]
We can fully test this after #441 is complete.
